ui/qt/src/Ariadne/UI/Qt/Widgets/StatusBar.hs

```diff
@@ -31,25 +31,27 @@ initStatusBar :: QMainWindow.QMainWindow -> IO StatusBar
 initStatusBar mainWindow = do
   statusBar <- QMainWindow.statusBar mainWindow
 
-  tipHeaderHashLabel <- QLabel.new
-  tipSlotLabel <- QLabel.new
-  currentSlotLabel <- QLabel.new
+  syncProgressLabel <- QLabel.newWithParent statusBar
+  blockchainLocalLabel <- QLabel.new
+  blockchainNetworkLabel <- QLabel.new
 
-  QStatusBar.addPermanentWidget statusBar tipHeaderHashLabel
-  QStatusBar.addPermanentWidget statusBar tipSlotLabel
-  QStatusBar.addPermanentWidget statusBar currentSlotLabel
+  QStatusBar.addPermanentWidget statusBar blockchainLocalLabel
+  QStatusBar.addPermanentWidget statusBar blockchainNetworkLabel
 
   return StatusBar{..}
 
 displayBlockchainInfo :: UiCardanoStatusUpdate -> UI StatusBar ()
 displayBlockchainInfo UiCardanoStatusUpdate{..} = do
-  tipHeaderHashLabel <- view tipHeaderHashLabelL
-  tipSlotLabel <- view tipSlotLabelL
-  currentSlotLabel <- view currentSlotLabelL
-
-  liftIO $ QLabel.setText tipHeaderHashLabel $ toString $
-    format ("Tip Hash: " % stext) tipHeaderHash
-  liftIO $ QLabel.setText tipSlotLabel $ toString $
-    format ("Tip slot: " % stext) tipSlot
-  liftIO $ QLabel.setText currentSlotLabel $ toString $
-    format ("Current slot: " % stext) currentSlot
+  StatusBar{..} <- ask
+  liftIO $ do
+    QLabel.setText blockchainLocalLabel $ toString $
+      format ("Local: " % stext) blockchainLocal
```

Comment:
Maybe while at it, ditch `format` from here and chage to `"Local: " <> blockchainLocal`, etc. I didn't know it's possible when I first wrote this.